migrator/migrations/m_187_to_m_188_add_nftables_to_policy/migration.go

```diff
@@ -0,0 +1,102 @@
+package m187tom188
+
+import (
+	"context"
+	"embed"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	"github.com/stackrox/rox/migrator/migrations/m_187_to_m_188_add_nftables_to_policy/schema"
+	"github.com/stackrox/rox/migrator/migrations/policymigrationhelper"
+	"github.com/stackrox/rox/migrator/types"
+	"gorm.io/gorm"
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: 187,
+		VersionAfter:   &storage.Version{SeqNum: 188},
+		Run: func(databases *types.Databases) error {
+			err := updatePolicies(databases.GormDB)
+			if err != nil {
+				return errors.Wrap(err, "updating policies")
+			}
+			return nil
+		},
+	}
+
+	//go:embed policies_before_and_after
+	policyDiffFS embed.FS
+
+	// TODO: fix comment and name
+	// Add policy exclusions only if the existing name, description and policy sections haven't changed.
+	fieldsToCompareForExclusions = []policymigrationhelper.FieldComparator{
+		policymigrationhelper.NameComparator,
+		policymigrationhelper.DescriptionComparator,
+		policymigrationhelper.PolicySectionComparator,
+	}
+
+	policyDiffs = []policymigrationhelper.PolicyDiff{
+		{
+			FieldsToCompare: fieldsToCompareForExclusions,
+			PolicyFileName:  "exec-iptables.json",
+		},
+		{
+			FieldsToCompare: fieldsToCompareForExclusions,
+			PolicyFileName:  "exec-iptables-root.json",
+		},
+	}
+)
+
+func updatePolicies(db *gorm.DB) error {
+
+	return policymigrationhelper.MigratePoliciesWithDiffsAndStore(
+		policyDiffFS,
+		policyDiffs,
+		// Check if policy with specified id exists
```

Comment:
This is virtually the same as the func below that does the get.  So I'm wondering if we even need the Count func as we can probably satisfy its purpose with the Get func (depending on how you wire the rest of it up).  Aside from that the usage of gorm appears to be correct to me.